### DISCUSSION TITLE
Quick comments from a potential new user on your VS Code Market Place page

### DISCUSSION THREAD

**Discussion by David-Else** [biomejs/biome#1094] [General]
Hi, the project looks amazing, but the PR and promotion could be a lot better. I had a quick look at https://marketplace.visualstudio.com/items?itemName=biomejs.biome&ssr=false#overview, and here is some feedback:

> The Visual Studio Code extension for Biome brings first-party support for Biome to VS Code and similar editors such as VSCodium.

What is Biome? What does it do? What languages does it work for? Why would I want to use it. These should be the first things on the page.

> 
    💾 Format on save
    💡 Inline suggestions with quick fixes
    🚧 Refactoring

Formatting with 97% accuracy to the industry standard Prettier, link to the prize that was won, on save is no big feature
Linting with all the recommended rules from eslint, typescript eslint etc... (does it include all the recommended?)
Refactoring? I see no mention of this feature on your website, does Biome actually do this, like the Martin Fowler book?

> By default, the extension will not start unless a biome.json file is present at the root of the workspace. 

I am running it in Helix and this is not true, was this not fixed? I didn't think it needed this any more?

> Thhis supports formatting on save out of the box.

`Thhis`... typos in the age of spell correction? This looks really bad to potential users. If spelling is not checked, is the code?

Have a look at all the top extensions, they all have exciting high resolution GIFS of the extensions in action.

Anyway, I think the project is amazing, and I know you just got going really with the new fork, hope this doesn't sound negative, I want this project to achieve massive success!

**Comment by ematipico** [comment]
Thank you @David-Else for your feedback. Would you like to make a PR to enhance the README? 

> I am running it in Helix and this is not true, was this not fixed? I didn't think it needed this any more?

There are two things to clarify:
1. The behaviour is dictated by the editor plugin, not the LSP. The editor plugin decides how and when to binary.
2. This behaviour went under different rewrites, and we are still talking about it. But I think we agreed on how the behaviour should be and we will work on it. To summarize, I think that the behaviour in Helix is the correct one. 

> hope this doesn't sound negative

Not at all, I think you gave us great feedback with a lot of positive reinforcements!

**Reply by David-Else** [reply]
> Thank you @David-Else for your feedback. Would you like to make a PR to enhance the README?

I may be able to in the future, but unfortunately I am too busy now. I would not advise waiting for me to update it.

I am waiting on https://github.com/biomejs/biome/issues/1042 till I can actually use Biome in Helix, I don't use VS Code myself, but I recognise that most people do and the editor plugin is important.

**Comment by ematipico** [comment]
@nhedger you might be interested in this good feedback :)
